### PR TITLE
fix(admin): unification of the "Install/Upgrade all" button in Module…

### PR DESCRIPTION
**Pull request by vhr**
…s/Widgets management page

Resolve MON-2865

### CODE REVIEW

File: www/include/options/oreon/widgets/list.php [hunk 1]
```diff
@@ -54,8 +54,7 @@
 $labels['author'] = _('Author');
 $labels['actions'] = _('Actions');
 
-$tpl->assign('hasWidgetsForInstallation', $widgetInfoObj->hasWidgetsForInstallation());
-$tpl->assign('hasWidgetsForUpgrade', $widgetInfoObj->hasWidgetsForUpgrade());
+$tpl->assign('canUpgradeOrInstallWidgets', $widgetInfoObj->hasWidgetsForInstallation() || $widgetInfoObj->hasWidgetsForUpgrade());
```
Comment (kduret) [inline www/include/options/oreon/widgets/list.php]: exceed 120 characters?

File: www/include/options/oreon/widgets/list.ihtml [hunk 2]
```diff
@@ -1,9 +1,10 @@
-{if $hasWidgetsForInstallation}
-    <img class="installAllBtn ico-16 margin_right" src="./img/icons/generate_conf.png" style="cursor: pointer;" title="Install all widgets">
-{/if}
-
-{if $hasWidgetsForUpgrade}
-    <img class="upgradeAllBtn ico-16" src="./img/icons/upgrade.png" style="cursor: pointer;" title="Upgrade all widgets">
+{if $canUpgradeOrInstallWidgets}
+    <br>
+	<a href="javascript:;" class="btc bt_success installAllBtn">
+		Install/Upgrade all
```
Comment (kduret) [inline www/include/options/oreon/widgets/list.ihtml]: need 4 spaces indentation

Comment (vhr) [reply www/include/options/oreon/widgets/list.ihtml]: done

File: www/include/options/oreon/widgets/list.php (hunk 1 above)
Comment (vhr) [reply www/include/options/oreon/widgets/list.php]: done

**Pull request merged**